tests/integration/ha/continuous_writes.py

```diff
@@ -56,6 +57,8 @@ async def start(self, repl_on_all_nodes: bool = False, is_bulk: bool = True) ->
         # create index if custom conf needed
         if repl_on_all_nodes:
             await self._create_fully_replicated_index()
+        else:
+            await self._create()
```

Comment:
If we let opensearch auto-create this index, it will create with "1-all" replicas; which for a test of scaling down 1 and back, won't work.

Reply:
Can we rename the method to something more along the lines of `_create_index_with_0_all_replicas`